api/tests/unit/organisations/test_unit_organisations_views.py

```diff
@@ -662,12 +668,13 @@ def test_when_subscription_is_cancelled_then_cancellation_date_set_and_alert_sen
     ):
         # Given
         cancellation_date = datetime.now(tz=UTC) + timedelta(days=1)
+        current_term_end = int(datetime.timestamp(cancellation_date))
```

Comment:
I'm curious as to why this had to change as part of this PR?

Reply:
Because the Chargebee API sends unix time in integer seconds, not milliseconds or microseconds. So to go through the API we need to fix the integer resolution to get through the serializer.